test/setup.js

```diff
@@ -12,15 +12,15 @@ function recordTypes() {
   }
 }
 
-before(() => {
+setup(() => {
   if (!GraphQLJSClient.startTracking) {
     return;
   }
 
   GraphQLJSClient.startTracking();
 });
 
-after(() => {
+teardown(() => {
```

Comment:
these aren't equivalent. If anything, skip them in the browser. `before` and `after` run before the test suit and after the test suite. `setup` and `teardown` run before _every_ test and after _every_ test. Maybe just wrap these up in an `if (before && after)`